src/EnergyPlus/UnitarySystem.hh

```diff
@@ -776,22 +777,22 @@ namespace UnitarySystems {
         );
 
         void calcUnitaryCoolingSystem(EnergyPlusData &state,
-                                      int const AirLoopNum,          // index to air loop
-                                      bool const FirstHVACIteration, // True when first HVAC iteration
-                                      Real64 const PartLoadRatio,    // coil operating part-load ratio
-                                      int const CompOn,              // compressor control (0=off, 1=on)
+                                      int const AirLoopNum,                        // index to air loop
+                                      bool const FirstHVACIteration,               // True when first HVAC iteration
+                                      Real64 const PartLoadRatio,                  // coil operating part-load ratio
+                                      DataHVACGlobals::CompressorOperation CompOn, // compressor control (0=off, 1=on)
```

Comment:
I removed const qualifiers in declarations, [suggestion from clang-tidy](https://clang.llvm.org/extra/clang-tidy/checks/readability-avoid-const-params-in-decls.html).